telco-examples/mgmt-cluster/multi-node/eib/mgmt-cluster-multinode.yaml

```
@@ -76,9 +76,9 @@ kubernetes:
       - name: jetstack
         url: https://charts.jetstack.io
       - name: rancher-charts
```

```suggestion
      - name: rancher
```

Must be edited to work with the changed repository names above.

---

changed